spec/config/mongoid.rb

```diff
@@ -2,12 +2,14 @@
   config.connect_to('delayed_job_shallow_mongoid_test')
 end
 
+Mongo::Logger.logger = ::Logger.new('tmp/mongo.log') if defined?(Mongo)
```

Comment:
I tend to agree that it would be better to do this via log levels than generating an additional file.